ios/Podfile

```diff
@@ -0,0 +1,70 @@
+# Uncomment the next line to define a global platform for your project
+platform :ios, '12.0'
+
+target 'NativeTemplate' do
+  # Comment the next line if you don't want to use dynamic frameworks
+  # use_frameworks!
+  
+
+  # Pods for nativeTemplate
+  # RN pods start
+  pod 'React', :path => '../node_modules/react-native', :subspecs => [
+    'Core',
```

Comment:
Does it make sense to order these alphabetically? Or is this some other meaningful order?

Reply:
I believe this was just lifted as in from the developer app

Reply:
Idd; for keeping it nice I will order them alphabetically.